src/components/base/button/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import propTypes from "prop-types";
+
+import Styles from "./index.module.scss";
+
+function Button({ children, onButtonClick, className, buttonType }) {
+  return (
+    <button
+      className={`${Styles.button} ${className}`}
```

Comment:
Could we assign a variable for classes ?
```suggestion
      className={classes}
```